apps/testingaccessibility/src/utils/get-resource-paths.ts

```diff
@@ -0,0 +1,37 @@
+import {SanityDocument} from '@sanity/client'
+import find from 'lodash/find'
+import isEmpty from 'lodash/isEmpty'
+
+export const getPathForLesson = (
+  lessonSlug: string,
+  modules: SanityDocument[],
+) => {
+  let path = {}
+  modules?.map((module: any) => {
```

Comment:
suggestion: it’d be great to see these “any” types get some actual typing

Reply:
how are these https://github.com/skillrecordings/products/pull/226/commits/d78e90ce92e8898296282391fa34a8ada8ed8c29 ?